packages/wonder-blocks-dropdown/src/components/single-select.md

```diff
@@ -477,7 +477,7 @@ class ExampleWithFilter extends React.Component {
     constructor() {
         super();
         this.state = {
-            opened: true,
+            opened: false,
```

Comment:
Made this change to avoid an issue with `Enzyme` (using gen-snapshots). This case is already covered by Chromatic.